netket/logging/runtime_log.py

```diff
@@ -46,3 +52,49 @@ def __getitem__(self, key):
 
     def flush(self, variational_state):
         pass
+
+    def serialize(self, path: str):
```

Comment:
```suggestion
    def serialize(self, path: str):
```
I'd slightly prefer if this would also be able to accept an IO object (like `outfile`). in case users want to serialize the runtime log data to something other than a file path. Maybe we could just accept both (i.e., use the present logic if `path` is an instance of `str` [and maybe `pathlib.Path` too, in order to support more modern Python features] and otherwise skip the `open` call).

Reply:
yes, I agree